### PR TITLE
Feature/stud 244/create team modal

### PR DESCRIPTION
**Pull request by yasfahm**
Create team modal shows up on CREATE TEAM button click
IMG_DFAA1B72A5BF-1

### CODE REVIEW

File: src/components/modals/CreateTeam.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    padding-horizontal: ${theme.SPACING_MEDIUM};
+`
+
+const ConfirmButton = styled.TouchableOpacity `
+    background-color: ${theme.COLOR_GREEN};
+    border-top-left-radius: 100;
+    border-bottom-left-radius: 100;
+    border-top-right-radius: 100;
+    border-bottom-right-radius: 100;
+    width: 30%;
```
Comment (car12lin12) [inline src/components/modals/CreateTeam.js]: can you make this 35%? the last letter for create and cancel break to newline for my smol phone

Comment (yasfahm) [reply src/components/modals/CreateTeam.js]: SAD

Comment (yasfahm) [reply src/components/modals/CreateTeam.js]: updated!

**Review by car12lin12 — APPROVED** [review]
nicely done Yas!

**Pull request merged**